src/docs/release-notes/R21/R21.mdx

```diff
@@ -0,0 +1,47 @@
+**Important**: we will be renamed to _Brightlayer User Interface_ (BLUI) soon, but our rapport to you all will stay the same 🤗
```

Comment:
```suggestion
**Important**: this will be the final release for PX Blue. For our next release, we will be renaming our platform to _Brightlayer User Interface_ (BLUI). Stay tuned for more details. 🤗
```